doctr/models/classification/textnetFast/pytorch.py

```diff
@@ -0,0 +1,285 @@
+    <https://arxiv.org/abs/2111.02394>`_.
+
+    >>> import torch
+    >>> from doctr.models import textnetfast_tiny
+    >>> model = textnetfast_tiny(pretrained=False)
+    >>> input_tensor = torch.rand((1, 3, 512, 512), dtype=torch.float32)
+    >>> out = model(input_tensor)
+
+    Args:
+        pretrained: boolean, True if model is pretrained
+
+    Returns:
+        A TextNet model
+    """
+
+    return _textnetfast(
+        "textnetfast_tiny",
+        pretrained,
+        TextNetFast,
+        stage1 = [ {"in_channels": 64, "out_channels": 64, "kernel_size": [3, 3], "stride": 1},
+                   {"in_channels": 64, "out_channels": 64, "kernel_size": [3, 3], "stride": 2},
+                   {"in_channels": 64, "out_channels": 64, "kernel_size": [3, 3], "stride": 1}],
+                   
+        stage2 = [ {"in_channels": 64, "out_channels": 128, "kernel_size": [3, 3], "stride": 2},
+                   {"in_channels": 128, "out_channels": 128, "kernel_size": [1, 3], "stride": 1},
+                   {"in_channels": 128, "out_channels": 128, "kernel_size": [3, 3], "stride": 1},
+                   {"in_channels": 128, "out_channels": 128, "kernel_size": [3, 1], "stride": 1},],
+                   
+        stage3 = [ {"in_channels": 128, "out_channels": 256, "kernel_size": [3, 3], "stride": 2},
+                   {"in_channels": 256, "out_channels": 256, "kernel_size": [3, 3], "stride": 1},
+                   {"in_channels": 256, "out_channels": 256, "kernel_size": [3, 1], "stride": 1},
+                   {"in_channels": 256, "out_channels": 256, "kernel_size": [1, 3], "stride": 1},],
+                   
+        stage4 = [ {"in_channels": 256, "out_channels": 512, "kernel_size": [3, 3], "stride": 2},
+                   {"in_channels": 512, "out_channels": 512, "kernel_size": [3, 1], "stride": 1},
+                   {"in_channels": 512, "out_channels": 512, "kernel_size": [1, 3], "stride": 1},
+                   {"in_channels": 512, "out_channels": 512, "kernel_size": [3, 3], "stride": 1}],
+                   
+        ignore_keys=["fc.weight", "fc.bias"],
+        **kwargs,
+    )
+    
+def textnetfast_small(pretrained: bool = False, **kwargs: Any) -> TextNetFast:
+    """TextNetFast architecture as described in `"FAST: Faster Arbitrarily-Shaped Text Detector with Minimalist Kernel Representation",
+    <https://arxiv.org/abs/2111.02394>`_.
+
+    >>> import torch
+    >>> from doctr.models import textnetfast_small
+    >>> model = textnetfast_small(pretrained=False)
+    >>> input_tensor = torch.rand((1, 3, 512, 512), dtype=torch.float32)
+    >>> out = model(input_tensor)
+
+    Args:
+        pretrained: boolean, True if model is pretrained
+
+    Returns:
+        A TextNetFast model
+    """
+
+    return _textnetfast(
+        "textnetfast_small",
+        pretrained,
+        TextNetFast,
+        stage1 = [ {"in_channels": 64, "out_channels": 64, "kernel_size": [3, 3], "stride": 1},
```

Comment:
@felixdittrich92 Maybe something with an external yaml file ? I mean a yaml file with basic configuration to initialize the architecture, so you can only tweak those files instead of modifying the code